manifests/cloud-provider-example.yaml

```diff
@@ -26,6 +26,11 @@ compartment: ocid1.compartment.oc1..aaaaaaaa3um2atybwhder4qttfhgon4j3hcxgmsvnyvx
 vcn: ocid1.vcn.oc1..aaaaaaaask7mpk4mij3pnm6yvnntte25ffadxiivpokxevfxgtsu6ftkqhrq
 
 loadBalancer:
+  # defaultSourceCIDRs configures the default permitted source CIDR range(s) for
+  # load balancers. Defaults to allow-all (0.0.0.0/0) if not provided.
+  defaultSourceCIDRs:
+    - 10.0.0.0/24
```

Comment:
I would use 0.0.0.0/0 as the example in case people copy the file and then don't understand why the LB is not serving traffic to their cluster.